src/components/Wedding.js

```diff
@@ -10,13 +10,16 @@ const Wedding = () => {
         <WeddingText>
           <EventDate>
             <EventLocation>
-              <h5>
+              <h2>Brooklyn, NY</h2>
+              <h5>Saturday, November 20, 2021</h5>
+              {/* <h5>
```

Comment:
Comments here needed?

Reply:
keeping it for now as I hope to refer back to this file soon to get the section done.